### PR TITLE
test against django>=2.2,<4.0

### PR DESCRIPTION
**Pull request by lampslave**

### CODE REVIEW

File: tests/testapp/tests/test_client.py [hunk 1]
```diff
@@ -32,8 +32,8 @@ class TestForm(forms.Form):
 
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-    url(r'^view/', view),
+    re_path(r'^admin/', admin.site.urls),
+    re_path(r'^view/', view),
```
Comment (GeyseR) [inline tests/testapp/tests/test_client.py]: what about replacing `re_path` with the `path` function?
there is nothing special that requires regular expression parsing

Comment (lampslave) [reply tests/testapp/tests/test_client.py]: I was too lazy to get rid of regular expressions :)

File: tox.ini [hunk 2]
```diff
@@ -1,26 +1,23 @@
 [tox]
 envlist =
-    py{35,36,37,38,39}-django22
-    py{36,37,38,39}-django30
-    py{36,37,38,39}-django31
+    py{36,37,38,39}-django{22,31,32}
```
Comment (GeyseR) [inline tox.ini]: I think it still makes sense to test django 2.2 against python 3.5, as it officially supported in this version https://docs.djangoproject.com/en/2.2/releases/2.2/#python-compatibility

Comment (lampslave) [reply tox.ini]: 3.5 is dead but ok.

**Pull request merged**